custom/icds_reports/templates/icds_reports/icds_app/mobile/awc-reports.html

```diff
@@ -0,0 +1,80 @@
+{% load staticfiles %}
```

Comment:
this is basically a direct import of this file https://github.com/dimagi/mobile-dashboard-prototype/pull/17/files#diff-f6e8c66fb33aef14103a9c901c4f54da

it completely changed through the course of this PR and ended up around only 20 functional lines so probably isn't worth reviewing here.